app/full_text_search.py

```diff
@@ -209,10 +252,10 @@ def create_romanized_name_fields(romanize_method, **kwargs):
     romanized_names_list = []
     romanize_method_name = romanize_method.__name__
     for field in kwargs:
-        romanized_name = romanize_method(kwargs[field])
-        if romanized_name:
+        romanized_names = romanize_method(kwargs[field])
```

Comment:
Not related to this pull request, but it's confusing that "field" and "fields" stores something different. Can you rename this "field" to "field_name" or something?